### ISSUE TITLE
consensus: Step doesn't increase iteration when quorum is reached

### ISSUE THREAD

**Issue by herr-seppia** [dusk-network/rusk#1255]
**Describe the bug**
Consensus doesn't stop when quorum is reached

**To Reproduce**
run testbed with 30 provisioners... first blocks will have this issue

**Expected behaviour**

**Logs/Screenshot**
```
885674 step=1 signature="845adb44de4ae8b5...4b85b9cbf511bb27"
2024-01-05T11:04:27.278690Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=50 target=43 bitset=83885675 step=1 signature="8117f4b415aa099f...2f0ffc30fd1de624"
2024-01-05T11:04:27.279856Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=42 target=43 bitset=3112856 step=2 signature="85ad7c16e74c042a...d78a351e15467f8c"
2024-01-05T11:04:27.281018Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=52 target=43 bitset=83885803 step=1 signature="8d5b76ac2d343c71...d016b46113ce3645"
2024-01-05T11:04:27.282171Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=47 target=43 bitset=3112888 step=2 signature="b42985b3c9b6837c...9e9b93ee5d5c2a18"
2024-01-05T11:04:27.289836Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=53 target=43 bitset=100663019 step=1 signature="84408f238c9b78e3...c99297333b3408cd"
2024-01-05T11:04:27.291071Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=55 target=43 bitset=100663275 step=1 signature="8bca292b5523e30b...324d68240a33db7a"
2024-01-05T11:04:27.293804Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=49 target=43 bitset=3112952 step=2 signature="95d2d6cd441467ff...4508de5c89602da9"
2024-01-05T11:04:27.310515Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=51 target=43 bitset=3112956 step=2 signature="a1c25deafc12e21b...fe76c577cfa9b05d"
2024-01-05T11:04:27.311700Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=60 target=43 bitset=100663291 step=1 signature="9194e6abcaf29387...a275c638b06817c1"
2024-01-05T11:04:27.365486Z  INFO main{round=5 iter=1 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="0000000000000000...0000000000000000" total=63 target=43 bitset=100663295 step=1 signature="833adfb34394dfa9...c9f8ea7a02fecc73"
2024-01-05T11:04:28.032050Z  INFO main{round=5 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}:voting{hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf"}: node::chain::consensus: verifying state
2024-01-05T11:04:28.032067Z  INFO main{round=5 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}:voting{hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf"}: rusk::vm: Received verify_state_transition request
2024-01-05T11:04:28.531274Z  INFO main{round=5 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf" total=45 target=43 bitset=81747960 step=4 signature="b8ad51ecbaa0942d...cb858bc758d89832"
2024-01-05T11:04:29.178813Z  INFO main{round=5 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf" total=46 target=43 bitset=81747962 step=4 signature="abdc689359b1b5dd...2592c4f901e46cd7"
2024-01-05T11:04:29.179204Z  INFO main{round=5 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf" total=47 target=43 bitset=98525178 step=4 signature="a8a19797bb439e31...2bbf0a7f816d6be9"
2024-01-05T11:04:29.270875Z  INFO main{round=5 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf" total=48 target=43 bitset=98533370 step=4 signature="98ddf17e4d561d1a...76d50d82ee6f58c5"
2024-01-05T11:04:29.271049Z  INFO main{round=5 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf" total=51 target=43 bitset=98566138 step=4 signature="993aa613a567c9e6...89e0cdf931ee65db"
2024-01-05T11:04:29.569449Z  INFO main{round=5 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf" total=54 target=43 bitset=98566142 step=4 signature="b46423954d8e40cd...7c4537e1e38ea149"
2024-01-05T11:04:29.587112Z  INFO main{round=5 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="c5805d3c7414f4f8...4d1395d8e4cb9ddf" total=56 target=43 bitset=98566143 step=4 signature="b2806781578331f4...deb3d7ea04604b7c"

```

**Additional context**

**Comment by herr-seppia** [comment]
Below the logs of https://github.com/dusk-network/rusk/pull/1256


```
2024-01-12T09:30:38.992512Z  INFO node::chain::consensus: event="spawn consensus" id=1 round=1 all=30 eligible=30
2024-01-12T09:30:43.993542Z  INFO main{round=1 iter=0 name=Proposal pk="pFPEcfxidLvwmFKR"}: dusk_consensus::execution_ctx: event="timeout-ed"
2024-01-12T09:30:58.992735Z  WARN node::databroker: error on handling msg: no blocks found
2024-01-12T09:30:58.993528Z  INFO node::chain::fsm: event="fsm::idle" height=0 iter=0 timeout_sec=20 finalized_height=0
2024-01-12T09:31:04.758182Z  INFO kadcast::maintainer: TableMaintainer::monitor_buckets back to sleep
2024-01-12T09:31:13.999546Z  INFO main{round=1 iter=0 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::execution_ctx: event="timeout-ed"
2024-01-12T09:31:14.104123Z ERROR main{round=1 iter=0 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::execution_ctx: phase handler err: InvalidValidation
2024-01-12T09:31:14.108003Z ERROR main{round=1 iter=0 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::execution_ctx: phase handler err: InvalidValidation
2024-01-12T09:31:14.111510Z ERROR main{round=1 iter=0 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::execution_ctx: phase handler err: InvalidValidation
2024-01-12T09:31:14.136690Z ERROR main{round=1 iter=0 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::execution_ctx: phase handler err: InvalidValidation
2024-01-12T09:31:18.995432Z  INFO node::chain::fsm: event="fsm::idle" height=0 iter=0 timeout_sec=20 finalized_height=0
2024-01-12T09:31:38.997529Z  INFO node::chain::fsm: event="fsm::idle" height=0 iter=0 timeout_sec=20 finalized_height=0
2024-01-12T09:31:58.999074Z  INFO node::chain::fsm: event="fsm::idle" height=0 iter=0 timeout_sec=20 finalized_height=0
2024-01-12T09:32:14.100136Z  INFO main{round=1 iter=0 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::execution_ctx: event="timeout-ed"
2024-01-12T09:32:15.085882Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.104023Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.240973Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.244635Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.248295Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.251939Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.255583Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.259231Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.262876Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.266521Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.270170Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.273823Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.283857Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.329200Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.431222Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.447385Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.453802Z  INFO main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="3181851938b8fdfb...42be1903030244e0" total=44 target=43 bitset=19775406 step=4 signature="9339e22bb6c3994c...16757e3db55b6672"
2024-01-12T09:32:15.453818Z ERROR main{round=1 iter=1 name=Validation pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.455043Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.614580Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.673163Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.776629Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.835263Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:15.995505Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.064537Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.137348Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.248232Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.319749Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.377121Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.472319Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.530739Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.553023Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.703163Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.853337Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:16.966804Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:17.085415Z  INFO main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::aggregator: event="quorum reached" hash="3181851938b8fdfb...42be1903030244e0" total=43 target=43 bitset=61210353 step=5 signature="979bb1b4968ffd1f...1f575dfd2c173d22"
2024-01-12T09:32:17.085439Z ERROR main{round=1 iter=1 name=Ratification pk="pFPEcfxidLvwmFKR"}: dusk_consensus::step_votes_reg: Attempting to replace [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] with [49, 129, 133, 25, 56, 184, 253, 251, 191, 174, 134, 116, 206, 31, 63, 227, 178, 82, 46, 88, 13, 226, 129, 253, 66, 190, 25, 3, 3, 2, 68, 224]
2024-01-12T09:32:17.086949Z  INFO node::chain: event="block received" src="wire" blk_height=1 blk_hash="3181851938b8fdfb...42be1903030244e0"
2024-01-12T09:32:17.087147Z  INFO node::chain::acceptor: event="verify_cert" cert_type="failed_cert" iter=0
2024-01-12T09:32:17.209876Z  INFO rusk::vm: Received accept request
2024-01-12T09:32:17.217855Z  WARN node::databroker: error on handling msg: could not find candidate
2024-01-12T09:32:17.238723Z  WARN node::chain::acceptor: event="missed iteration" height=1 iter=0 generator="rdHa4H5p1rVtQyqD"
2024-01-12T09:32:17.238768Z  INFO node::chain::acceptor: event="block accepted" height=1 iter=1 hash="3181851938b8fdfb...42be1903030244e0" txs=0 state_hash="76a999f513a5ffa3...a71596a7477bfdcd" fsv_bitset=27639726 ssv_bitset=128319201 block_time=58209711 generator="m6dy2gz3jC3ifLpZ" dur_ms=28 label="Accepted" final_from_rolling=false
2024-01-12T09:32:17.238782Z  INFO node::chain::consensus: event="spawn consensus" id=2 round=2 all=30 eligible=30
```

**Comment by herr-seppia** [comment]
Resolved with #1831
